hw-model/src/lib.rs

```diff
@@ -248,6 +254,9 @@ pub struct BootParams<'a> {
     pub initial_dbg_manuf_service_reg: u32,
     pub initial_repcnt_thresh_reg: Option<CptraItrngEntropyConfig1WriteVal>,
     pub initial_adaptp_thresh_reg: Option<CptraItrngEntropyConfig0WriteVal>,
+    #[cfg(feature = "fpga_realtime")]
```

Comment:
Can we just make this a vector of valid pausers? (no conditional computation?)

Reply:
Makes sense. Will do.

Reply:
Done